examples/pyfluent-workflow/main.py

```diff
@@ -0,0 +1,220 @@
+hot_inlet.momentum.velocity.value = 1.2
+hot_inlet.turbulence.turbulent_specification = "Intensity and Hydraulic Diameter"
+hot_inlet.turbulence.turbulent_intensity = 0.05
+hot_inlet.turbulence. hydraulic_diameter = "1 [in]"
+hot_inlet.thermal.t.value = 313.15
+
+# - pressure outlet (outlet), Setting: Value:
+# - Backflow Turbulent Intensity: 5 [%]
+# - Backflow Turbulent Viscosity Ratio: 4
+
+fluent_session.setup.boundary_conditions.pressure_outlet["outlet"].turbulence.turbulent_viscosity_ratio_real = 4
+
+# ## Initialize flow field
+
+fluent_session.solution.initialization.hybrid_initialize()
+
+# ## Solve for 150 iterations
+# Setting iteration count to 150 to solve the model.
+
+fluent_session.solution.run_calculation.iterate(iter_count=150)
+
+# ## Update Solution using Workbench Journal Commands
+
+wb.run_script_string("system1 = GetSystem(Name=\"FLU\")")
+wb.run_script_string("solutionComponent1 = system1.GetComponent(Name=\"Solution\")")
+wb.run_script_string("solutionComponent1.Update(AllDependencies=True)")
+
+# ## Postprocessing
+# Create and display velocity vectors on the ``symmetry-xyplane`` plane.
+
+# ## Configure graphics picture export
+# Since Fluent is being run without the GUI, you must to export plots as picture files. Edit the picture settings to use a custom resolution so that the images are large enough.
+
+graphics = fluent_session.results.graphics
+if graphics.picture.use_window_resolution.is_active():
+    graphics.picture.use_window_resolution = False
+graphics.picture.x_resolution = 1920
+graphics.picture.y_resolution = 1440
+
+# ## Create velocity vectors
+# Create and display velocity vectors on the ``symmetry-xyplane`` plane. Then, export the image for inspection.
+
+graphics = fluent_session.results.graphics
+
+graphics.vector["velocity_vector_symmetry"] = {}
+velocity_symmetry = fluent_session.results.graphics.vector["velocity_vector_symmetry"]
+velocity_symmetry.print_state()
+velocity_symmetry.field = "velocity-magnitude"
+velocity_symmetry.surfaces_list = [
+    "symmetry-xyplane",
+]
+velocity_symmetry.scale.scale_f = 4
+velocity_symmetry.style = "arrow"
+velocity_symmetry.display()
+
+graphics.views.restore_view(view_name="front")
+graphics.views.auto_scale()
+graphics.picture.save_picture(file_name="velocity_vector_symmetry.png")
+
+# ## Compute mass flow rate
+# Compute the mass flow rate.
+
+# +
+fluent_session.solution.report_definitions.flux["mass_flow_rate"] = {}
+
+mass_flow_rate = fluent_session.solution.report_definitions.flux["mass_flow_rate"]
+mass_flow_rate.boundaries = [
+    "cold-inlet",
+    "hot-inlet",
+    "outlet",
+]
+mass_flow_rate.print_state()
+fluent_session.solution.report_definitions.compute(report_defs=["mass_flow_rate"])
+# -
+
+# ## Save project
+
+file_path = os.path.join(server_workdir.absolute(), "mixing_elbow.wbpj")
+save_string = "Save(FilePath=\"" + file_path + "\"," + "Overwrite=True)"
+wb.run_script_string(save_string)
```

Comment:
If you run a script on the server, you can always query the server's working directory in the script itself:
```
save_string = """workdir = GetServerWorkingDirectory()
Save(FilePath=workdir + "mixing_elbow.wbpj", Overwrite=True)"""
```